src/containers/meeting/third-party/Detail/index.tsx

```
@@ -1,58 +1,253 @@
+
+      await GroupService.updateGroupSchedule(client, input);
+
+      await fetchSchedule();
+
+      showSuccess('일정이 수정되었습니다.');
+
+      setTimeout(() => {
+        router.replace('/meeting/third-party?refresh=true');
+      }, 1000);
+
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)